server/service/src/sync/translations/stock_line.rs

```diff
@@ -183,4 +203,43 @@ mod tests {
             assert_eq!(translation_result, record.translated_record);
         }
     }
+
+    #[actix_rt::test]
+    async fn test_stock_line_push_merged() {
+        // The item_links_merged function will merge ALL items into item_a, so all stock_lines should have an item_id of "item_a" regardless of their original item_id.
+        let (_, connection, _, _) = setup_all(
+            "test_stock_line_push_item_link_merged",
+            MockDataInserts::none()
+                .units()
+                .items()
+                .item_links_merged()
```

Comment:
I don't think you've actaully added any item_links_merged.

Btw would prefer new data to test with:

https://github.com/msupply-foundation/open-msupply/compare/2626-item-merge-sync-push-translators-item_id...2626-item-merge-sync-push-translators-item_id(New-mock-data)

Reply:
`item_links_merged()` actually just merges ALL items into `item_a` (i.e. updates all the item_links from all items), not making any new item_links. Bit overkill maybe but doesn't seem crazy - there's still only about a dozen items in the mock data. The advantage is not having to copy paste mock data like you suggest to all the other translator tests.

Why do we have the base mock data if I'm not to use it?

Reply:
I didn't see the logic of what item_links_merged, was kind of assuming it would insert some mock data like the rest of of MockDataInserts. 
I am completely fine with base mock data. But I think a method like this may need to be separated from MockDataInserts